pkg/reconciler/shared/tektonconfig/tektonconfig.go

```diff
@@ -137,6 +151,12 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, tc *v1alpha1.TektonConfi
 		}
 	}
 
+	// reconcile pruner installerSet
```

Comment:
if we need only if there is something specified in pruner field, otherwise not

Reply:
we do not have separate CR for pruner. still it is possible by having pruner section hash value.

I do not know how to track, if pruner RBAC/SA `installerset` get deleted. hence reconciling on each call.

Reply:
we can ccheck if config. spec.pruner is not nil

Reply:
Sure, I will update the code to skip pruner `installerset` reconcile, `if config. spec.pruner == nil`

Reply:
@piyush-garg I have updated the code to skip reconcile pruner installerset